libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm

```diff
@@ -13,3 +13,7 @@ SERIAL5_PROTOCOL 1
 # lower heater gains
 BRD_IMUHEAT_I 0.07
 BRD_IMUHEAT_P 50
+
+# compass primary defaults
+COMPASS_PRIO1_ID 73475
```

Comment:
This should be
```
COMPASS_PRIO1_ID COMPASS_DEV_ID2
COMPASS_PRIO2_ID COMPASS_DEV_ID
```

Reply:
No as per me, it should be the value associated with the particular compass_dev_id that we want at a certain priority

Reply:
using the string won't work, as the parser doesn't know how to decode parameter names in the right hand column

Reply:
Thank you for the clarification, @tridge !